packages/paste-core/components/alert-dialog/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@twilio-paste/alert-dialog",
+  "version": "0.0.0",
+  "category": "overlay",
+  "status": "production",
+  "description": "An alert dialog displays content over the main page and blocks any interaction with the page.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "main:dev": "src/index.tsx",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "yarn clean && NODE_ENV=production node build.js && tsc",
+    "build:js": "NODE_ENV=development node build.js",
+    "build:props": "typedoc --tsconfig ./tsconfig.json --json ./dist/prop-types.json",
+    "clean": "rm -rf ./dist",
+    "tsc": "tsc"
+  },
+  "peerDependencies": {
+    "@twilio-paste/box": "^4.0.2",
+    "@twilio-paste/modal": "^7.0.1",
+    "@twilio-paste/theme": "^5.0.1",
+    "@twilio-paste/types": "^3.1.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
+  },
+  "devDependencies": {
+    "@twilio-paste/box": "^4.0.5",
+    "@twilio-paste/modal": "^7.0.1",
+    "@twilio-paste/theme": "^5.1.0",
+    "@twilio-paste/types": "^3.1.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6"
```

Comment:
We'll need to add heading, button, stack, and uid-library to peerDeps and devDeps since we're using them in the component.

Reply:
Ah of course! Thanks for catching that. Where is it that uid-library is used?

Reply:
That library is generating the `headingID` and `bodyID` here: https://github.com/twilio-labs/paste/pull/1484/files#diff-33645d3572edd38a41d0c48139f80384adde5f8f86122ad2747ab42d407774bfR31-R32